src/tool/core/generate.rs

```diff
@@ -39,7 +39,6 @@ fn get_slug_from_args(specific_problem: &String) -> anyhow::Result<Cow<'_, Strin
         Cow::Owned(slug)
     } else {
         // This is expected to be a valid slug
-        //TODO: Validate slug
```

Comment:
sneakly removing this?
did we add this to change the logic of url ??

Reply:
fine just close this comment.

Reply:
Lol the commit message has an explanation